recipes/index.js

```diff
@@ -1,295 +1,302 @@
 #! /usr/bin/env node
 
-const program = require("commander");
-const inquirer = require("inquirer");
-const chalk = require("chalk");
-const shell = require("shelljs");
+const program = require('commander')
+const inquirer = require('inquirer')
+const chalk = require('chalk')
+const shell = require('shelljs')
 
-const slimConfig = require("./slim/config");
-const slimSnippet = require("./slim/snippets");
-const basicConfig = require(`./basic/config`);
-const basicSnippet = require(`./basic/snippets`);
+const slimConfig = require('./slim/config')
+const slimSnippet = require('./slim/snippets')
+const basicConfig = require(`./basic/config`)
+const basicSnippet = require(`./basic/snippets`)
 
 const {
   log,
   error,
   createFile,
   tryAccess,
   moduleSetInstall,
-} = require("./utils");
+} = require('./utils')
 
-program.option("-f, --app-name <value>", "App Name");
+program.option('-f, --app-name <value>', 'App Name')
 
-program.parse(process.argv);
+program.parse(process.argv)
 
 // Create App Directory.
-const appName = program.args[0];
+const appName = program.args[0]
 
 if (!appName) {
-  error("App name is mandatory to create your boiler plate");
+  console.error('Please specify the app name:')
+  // multiline erro - do not change the indentation
```

Comment:
it has typo `erro`. can we remove this comment. this comment not required when we have below line function